website/docs/integrations/mixpanel.mdx

```diff
@@ -0,0 +1,206 @@
+    )
+)
+```
+</TabItem>
+
+<TabItem value="ruby" label="Ruby">
+```ruby
+def on_flag_evaluated(evaluation_details):
+    mp.track(evaluation_details.user.get_identifier(), '$experiment_started',  {
+        'Experiment name': evaluation_details.key,
+        'Variant name': evaluation_details.value,
+        'Variation ID': evaluation_details.variation_id
+    })
+end
+
+client = ConfigCat.get('#YOUR-SDK-KEY#',
+    ConfigCat::ConfigCatOptions.new(
+        hooks: ConfigCat::Hooks.new(on_flag_evaluated: method(:on_flag_evaluated))
+    )
+)
+```
+</TabItem>
+
+<TabItem value="go" label="Go">
+```go
+client := configcat.NewCustomClient(configcat.Config{SDKKey: "#YOUR-SDK-KEY#",
+    Hooks: &configcat.Hooks{OnFlagEvaluated: func(details *EvaluationDetails) {
+        err := mp.Track(ctx, []*mixpanel.Event{
+            mp.NewEvent("$experiment_started", details.Data.User.(*UserData).Identifier, map[string]any{
+                "Experiment name": details.Data.Key,
+                "Variant name": details.Value,
+                "Variation ID": details.Data.VariationID,
+            }),
+        })
+    }}})
+```
+</TabItem>
+
+<TabItem value="java" label="Java">
+```java
+ConfigCatClient client = ConfigCatClient.get("#YOUR-SDK-KEY#", options -> {
+    options.hooks().addOnFlagEvaluated(details -> {
+        JSONObject props = new JSONObject();
+        props.put("Experiment name", details.getKey());
+        props.put("Variant name", details.getValue());
+        props.put("Variation ID", details.getVariationId());
+        JSONObject event = messageBuilder.event(details.getUser().getIdentifier(), "$experiment_started", props);
+
+        ClientDelivery delivery = new ClientDelivery();
+        delivery.addMessage(event);
+        mixpanel.deliver(delivery);
+    });
+});
+```
+</TabItem>
+
+<TabItem value="android" label="Android">
+```java
+ConfigCatClient client = ConfigCatClient.get("#YOUR-SDK-KEY#", options -> {
+    options.hooks().addOnFlagEvaluated(details -> {
+        JSONObject props = new JSONObject();
+        props.put("Experiment name", details.getKey());
+        props.put("Variant name", details.getValue());
+        props.put("Variation ID", details.getVariationId());
+        mp.track("$experiment_started", props);
+    });
+});
+```
+</TabItem>
+
+<TabItem value="ios" label="Swift (iOS)">
+```swift
+let client = ConfigCatClient.get(sdkKey: "#YOUR-SDK-KEY#") { options in
+    options.hooks.addOnFlagEvaluated { details in
+        Mixpanel.mainInstance().track(event:"$experiment_started", properties: [
+            "Experiment name": details.key,
+            "Variant name": details.value,
+            "Variation ID": details.variationId,
+        ])
+    }
+}
+```
+</TabItem>
+
+</Tabs>
+
+:::note
+If you are using Text feature flags with long text values (e.g. JSON values) that makes no sense to send into Mixpanel as a Variant, you can send the `variationId` as the `Variant name` to Mixpanel instead of the `value`.
```

Comment:
Anything positive would be better instead of 'makes no sense':
- recommended to do this instead of that
- it might be a good idea to do this rather than that

Reply:
removed the makes sense part